src/Cli/dotnet/commands/dotnet-add/AddCommandParser.cs

```diff
@@ -13,7 +13,8 @@ internal static class AddCommandParser
 
         public static readonly CliArgument<string> ProjectArgument = new CliArgument<string>(CommonLocalizableStrings.ProjectArgumentName)
         {
-            Description = CommonLocalizableStrings.ProjectArgumentDescription
+            Description = CommonLocalizableStrings.ProjectArgumentDescription,
```

Comment:
I think I do this in one of my PRs, so conflict, but that doesn't matter too much.